<launch>
    <arg name="color_height" default="480"/>
    <arg name="color_width" default="640"/>
    <arg name="depth_height" default="480"/>
    <arg name="depth_width" default="640"/>
    <arg name="align" default="false"/>
    <node
            pkg="realsense_video_publisher"
            type="realsense_video_topic_publisher_script.py"
            name="realsense_video_topic_publisher_node"
            output="screen">
        <param name="color_height" value="$(arg color_height)"/>
        <param name="color_width" value="$(arg color_width)"/>
        <param name="depth_height" value="$(arg depth_height)"/>
        <param name="depth_width" value="$(arg depth_width)"/>
        <param name="align" value="$(arg align)"/>
    </node>


</launch>